quarkus/integration-tests/pom.xml

```diff
@@ -74,7 +74,7 @@
         <profile>
             <id>native</id>
             <modules>
-               <module>server</module>
```

Comment:
So we aren't testing the server module anymore? Should we even have it in 15.1 then? Also any reason embedded also doesn't build with native?

Reply:
I don't want to remove code in a patch release (as was done in `main`).